optimum/exporters/onnx/model_patcher.py

```diff
@@ -509,6 +509,24 @@ def patched_forward(*args, **kwargs):
         self.patched_forward = patched_forward
 
 
+class ColPaliModelPatcher(ModelPatcher):
+    def __init__(
+        self,
+        config: "OnnxConfig",
+        model: Union["PreTrainedModel", "TFPreTrainedModel"],
+        model_kwargs: Optional[Dict[str, Any]] = None,
+    ):
+        super().__init__(config, model, model_kwargs)
+
+        def patched_forward(input_ids=None, pixel_values=None, attention_mask=None, **kwargs):
+            outputs = self.orig_forward(
+                input_ids=input_ids, pixel_values=pixel_values, attention_mask=attention_mask, **kwargs
+            )
+            return outputs
+
+        self.patched_forward = patched_forward
```

Comment:
why is it needed ?

Reply:
its needed because the original ColPali Model only takes **kwargs and no named arguments. This resulted in an error. This fixes that error